### PR TITLE
🏗 Fix Windows path resolution for Bento remap

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

**Review by danielrozenberg — APPROVED** [review]

File: build-system/compile/bento-remap.js [hunk 1]
```diff
@@ -6,21 +6,20 @@ const {once} = require('../common/once');
...
  */
 function resolveExactModuleFile(path) {
-  let unaliased = path.startsWith('#')
-    ? resolvePath(path)
-    : formatPathLikeSource(path);
+  let unaliased = path.startsWith('#') ? resolvePath(path) : path;
   try {
     if (lstatSync(unaliased).isDirectory()) {
-      unaliased += '/index';
+      unaliased = '/index';
```
Comment (danielrozenberg) [inline build-system/compile/bento-remap.js:22]: Why this change?

Comment (alanorozco) [reply build-system/compile/bento-remap.js:22]: shouldn't be there, my bad!
